tests/mypy/outputs/plugin-fail-strict.txt

```diff
@@ -15,7 +15,7 @@
 87: error: Missing named argument "c" for "DefaultTestingModel"  [call-arg]
 87: error: Missing named argument "d" for "DefaultTestingModel"  [call-arg]
 87: error: Missing named argument "e" for "DefaultTestingModel"  [call-arg]
-91: error: Name 'Undefined' is not defined  [name-defined]
+91: error: Name "Undefined" is not defined  [name-defined]
```

Comment:
Ah sorry I bumped all the deps together yesterday including mypy. Can you please rebase/merge?

Reply:
should be squared away now in https://github.com/samuelcolvin/pydantic/pull/2908/commits/c7491208644dfa8f3389882ca236ef9ccca1f7e6